### ISSUE TITLE
Migrate from ESH to OH

### ISSUE THREAD

**Issue by seime** [kokuwaio/esh-constants-maven-plugin#118]
Eclipse Smarthome is archived, and Openhab is on its own again.

The code generator generates code with references to Eclipse Smarthome packages. This would need to change to Openhab packages.

**Comment by fabian-schlegel** [comment]
Yep, that's true. The projects needs some work anyway. I hope to get this done in March.
